commit 31ca3de72f35f9ead7596ad104e29e9d912f3ac7
Merge: 1977b44 64139f5
Author: Bo Herrmannsen <bo.herrmannsen@gmail.com>
Date:   Wed Dec 17 21:42:53 2014 +0100

    Merge pull request #819 from neildarlow/Marlin_v1
    
    Activate LiquidTWI2 device detection and space-pad LCD status line.

diff --git a/Marlin/Marlin_main.cpp b/Marlin/Marlin_main.cpp
index 616b7a1..e8d800e 100644
--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -1896,6 +1896,8 @@ void process_commands()
         }
         lcd_ignore_click(false);
       }else{
+          if (!lcd_detected())
+            break;
         while(!lcd_clicked()){
           manage_heater();
           manage_inactivity();

